assignment1/numpy.ipynb

```diff
@@ -134,16 +134,16 @@
    },
    "outputs": [],
    "source": [
-    "# creates an vector with elements [0, 1, 2, 3, 4, 5]\n",
+    "# creates an vector with elements [0, 1, 2, 3, 4]\n",
     "a = np.arange(5)\n",
     "print(\"a = {}\".format(a))\n",
     "\n",
-    "# we can also set its starting point. So to create vector [10, 11, 12, 13, 14, 15]\n",
+    "# we can also set its starting point. So to create vector [10, 11, 12, 13, 14]\n",
     "v = np.arange(10, 15)\n",
     "print(\"v = {}\".format(v))\n",
     "\n",
     "# and also we can set step size. So to create a vector with elements spaced by 10\n",
-    "# and starts at 10 and ends at 50\n",
+    "# and starts at 10 and ends at 40\n, that is [10, 20, 30, 40]",
```

Comment:
@peto2006 Sadly, this does not seem correct (see the attached image), but I'll merge this PR and fix that separately.

Thanks for your help!
![screenshot from 2017-10-07 11-12-20](https://user-images.githubusercontent.com/461491/31306456-a76adc36-ab50-11e7-8ac8-5ec21d510741.png)